src/Samples/VisualBasic/ConsoleClassifier/Program.vb

```diff
@@ -21,10 +21,10 @@ Sub Main()
 WriteLine(""Hello, World!"")
 End Sub
 End Module")
-        document = Await Formatter.FormatAsync(document)
-        Dim text As SourceText = Await document.GetTextAsync()
+        document = Await Formatter.FormatAsync(document).ConfigureAwait(False)
```

Comment:
This shouldn't be running on samples though. We do it as a library so other apps don't have to think about it.